agir/groups/components/groupPage/GroupSettings/GroupMemberFile/GroupMemberActions.js

```diff
@@ -0,0 +1,112 @@
+import PropTypes from "prop-types";
+import React from "react";
+import styled from "styled-components";
+
+import Button from "@agir/front/genericComponents/Button";
+
+import { MEMBERSHIP_TYPES } from "@agir/groups/utils/group";
+
+const StyledWrapper = styled.div`
+  padding: 0;
+  margin: 0;
+  list-style-position: inside;
+
+  h4 {
+    font-size: 1rem;
+    margin: 0;
+    line-height: 1.5;
+    font-weight: 600;
+  }
+
+  p {
+    margin: 0;
+
+    ${Button} {
+      margin: 0.5rem 0.5rem 0 0;
+    }
+  }
+`;
+
+const GroupMemberFacts = (props) => {
+  const { isReferent, onChangeMembershipType, currentMembershipType } = props;
+
+  if (!onChangeMembershipType) {
+    return null;
+  }
+
+  if (currentMembershipType == MEMBERSHIP_TYPES.FOLLOWER) {
+    const handleClick = () => {
+      onChangeMembershipType(MEMBERSHIP_TYPES.MEMBER);
+    };
+
+    return (
+      <StyledWrapper>
+        <h4>Modifier les droits</h4>
+        <p>
+          <Button onClick={handleClick}>Passer en membre actif</Button>
+        </p>
+      </StyledWrapper>
+    );
+  }
+
+  if (currentMembershipType == MEMBERSHIP_TYPES.MEMBER) {
+    const setAsFollower = () => {
+      onChangeMembershipType(MEMBERSHIP_TYPES.FOLLOWER);
+    };
+    const setAsManager = () => {
+      onChangeMembershipType(MEMBERSHIP_TYPES.MANAGER);
+    };
+    return (
+      <StyledWrapper>
+        <h4>Modifier les droits</h4>
+        <p>
+          <Button onClick={setAsFollower}>Passer en contact</Button>
+          <Button onClick={setAsManager}>Passer en gestionnaire</Button>
+        </p>
+      </StyledWrapper>
+    );
+  }
+
+  if (isReferent && currentMembershipType == MEMBERSHIP_TYPES.MANAGER) {
+    const handleClick = () => {
+      onChangeMembershipType(MEMBERSHIP_TYPES.MEMBER);
+    };
+
+    return (
+      <StyledWrapper>
+        <h4>Modifier les droits</h4>
+        <p>
+          <Button onClick={handleClick}>
+            Retirer le droit de gestionnaire
+          </Button>
+        </p>
+      </StyledWrapper>
+    );
+  }
+
+  switch (currentMembershipType) {
+    case MEMBERSHIP_TYPES.MEMBER:
+      return (
+        <StyledWrapper>
+          <h4>Modifier les droits</h4>
+        </StyledWrapper>
+      );
+    case MEMBERSHIP_TYPES.MANAGER:
+      return (
+        <StyledWrapper>
+          <h4>Modifier les droits</h4>
+        </StyledWrapper>
+      );
```

Comment:
```suggestion
    case MEMBERSHIP_TYPES.MEMBER || MEMBERSHIP_TYPES.MANAGER:
      return (
        <StyledWrapper>
          <h4>Modifier les droits</h4>
        </StyledWrapper>
      );
```
Et du coup pourquoi pas
```
if ([MEMBERSHIP_TYPES.MEMBER, MEMBERSHIP_TYPES.MANAGER].includes(currentMembershipType) {
      return (
        <StyledWrapper>
          <h4>Modifier les droits</h4>
        </StyledWrapper>
      );
}
return null;
```